c2corg_ui/templates/base.html

```diff
@@ -25,10 +25,10 @@
   <body>
 
     <div id="page-header">
-    <a href="${request.route_url('index')}" title="Back to homepage"><img src="${request.static_url('c2corg_ui:static/img/logo.png')}" alt="Camptocamp.org" /></a>
+    <a href="${request.route_url('index')}" title="{{'Back to homepage'|translate}}"><img src="${request.static_url('c2corg_ui:static/img/logo.png')}" alt="Camptocamp.org" /></a>
     <ul>
-      <li><a href="${request.route_url('waypoints_index')}">Waypoints</a></li>
-      <li><a href="${request.route_url('routes_index')}">Routes</a></li>
+      <li><a href="${request.route_url('waypoints_index')}" translate>Waypoints</a></li>
```

Comment:
Why? This is a standard link, not an expression. And it's better to use HTML links so that search engines may access those pages.